src/QMCWaveFunctions/SPOSet.h

```diff
@@ -139,7 +139,10 @@ class SPOSet: public QMCTraits
     return OrbitalSetSize;
   }
 
-
+  inline bool hasIonDerivs() const
```

Comment:
This is a good move, please mark ionDerivs protected.
```
  ///flag to calculate ionic derivatives
  bool ionDerivs;
```
I think the meaning of this flag should be
```
  ///true, if the derived class has non-zero ionic derivatives.
```
Please add a consistent comment of this function as well.

Reply:
Will do!  Thanks!

Reply:
Please delete the setIonDerivs.